### ISSUE TITLE
Overlapping input and output variables in Go, Postgres, pgx/v4

### ISSUE THREAD

**Issue by kevinawoo** [sqlc-dev/sqlc#1559]
### Version

v1.13.0 and [6e45a9f](https://github.com/kyleconroy/sqlc/commit/6e45a9f608f869749aa2122ff1b9a74af109af73)

### What happened?

Almost the exactly like #1290, it seems like `"sql_package": "pgx/v4"` is the cause.

The generated Go code redeclares `id` on line 3:

```go
func (q *Queries) GetUser(ctx context.Context, id string) (string, error) {
	row := q.db.QueryRow(ctx, getUser, id)
	var id string
	err := row.Scan(&id)
	return id, err
}
```

### Database schema

```sql
-- Example queries for sqlc
CREATE TABLE public.users (
    id character varying(64) NOT NULL
);
```

### SQL queries

```sql
-- name: GetUser :one
SELECT *
FROM users
WHERE id = $1
LIMIT 1;
```

### Configuration

```yaml
{
  "version": "1",
  "packages": [
    {
      "path": "db",
      "engine": "postgresql",
      "schema": "query.sql",
      "queries": "query.sql",
      "sql_package": "pgx/v4"
    }
  ]
}
```

### Playground URL

https://play.sqlc.dev/p/07a36257aa58677adfa4d2558864d0a09ccc286c487893d5854978684b9babe5

### What operating system are you using?

macOS

### What database engines are you using?

PostgreSQL

### What type of code are you generating?

Go

**Comment by zaneli** [comment]
Seems to fixed by #1298

**Comment by kyleconroy** [comment]
Fixed! https://play.sqlc.dev/p/d0feca86867bf101295c1429f1d518f432fd6f6e6381d84daefb8c870a5776cc
